opensuse-kubectl-docker/Dockerfile

```diff
@@ -19,8 +17,11 @@ RUN true \
     curl \
     wget \
     tar \
-    helm \
     direnv \
+    aaa_base bash-completion docker-bash-completion systemd-bash-completion \
```

Comment:
Ah, I had assume that `aaa_base` would already been installed (based on its name), but that appears wrong.